tools/editor/multi_node_edit.cpp

```diff
@@ -52,8 +52,16 @@ bool MultiNodeEdit::_set(const StringName& p_name, const Variant& p_value){
 		Node*n=es->get_node(E->get());
 		if (!n)
 			continue;
+		else if (p_value.get_type() == Variant::NODE_PATH) {
+			Node *tonode = n->get_node(p_value);
+			if (tonode) {
+				NodePath p_path = n->get_path_to(tonode);
+				ur->add_do_property(n,name,p_path);
+			}
```

Comment:
I checked `Node::get_path_to` and it has fail conditions inside, `ERR_FAIL_NULL_V(p_node,NodePath())` etc. so if `tonode` isn't reachable for some reason it will fail so there's no need to add an extra error here, at least I think so... this patch should work as expected